docs/finality-provider-operation.md

```diff
@@ -0,0 +1,732 @@
+- `--security-contact`: Contact email for security issues 
+- `--details`: Additional description of your finality provider 
+- `--daemon-address`: RPC address of the finality provider daemon
+  (default: `127.0.0.1:12581`)
+
+Upon successful creation, the command will return a JSON response containing
+your finality provider's details:
+
+``` json
+{
+    "fp_addr": "bbn1ht2nxa6hlyl89m8xpdde9xsj40n0sxd2f9shsq", 
+    "eots_pk_hex":
+    "cf0f03b9ee2d4a0f27240e2d8b8c8ef609e24358b2eb3cfd89ae4e4f472e1a41",
+      "description": 
+      {
+        "moniker": "MyFinalityProvider", 
+        "website": "https://myfinalityprovider.com", 
+        "security_contact": "security@myfinalityprovider.com", 
+        "details": "finality provider for the Babylon network"
+      }, 
+      "commission": "0.050000000000000000", 
+      "status": "CREATED"
+} 
+```
+
+The response includes: 
+- `fp_addr`: Your Babylon account address for receiving
+  rewards 
+- `eots_pk_hex`: Your unique BTC public key identifier
+- `description`: Your finality provider's metadata 
+- `commission`: Your set commission rate 
+- `status`: Current status of the finality provider.
+
+
+Below you can see a list of the statuses that a finality provider can transition
+to:
+- `CREATED`: defines a finality provider that is awaiting registration
+- `REGISTERED`: defines a finality provider that has been registered
+  to the consumer chain but has no delegated stake
+- `ACTIVE`: defines a finality provider that is delegated to vote
+- `INACTIVE`: defines a finality provider whose delegations are reduced to 
+  zero but not slashed
+- `JAILED`: defines a finality provider that has been jailed
+
+For more information on statuses please refer to diagram in the core documentation 
+[fp-core](./fp-core).
+
+### 5.2. Register Finality Provider
+
+The `register-finality-provider` command registers your finality provider on the
+Babylon chain. This command requires you to specify:
+
+1. The EOTS public key of the finality provider you wish to register
+2. The Babylon account associated with your finality provider
+   (the one specified in the creation) having sufficient funds 
+   to pay for the transaction fee.
+3. A running fpd daemon
+
+``` shell
+fpd register-finality-provider \
+  <fp-eots-pk-hex> \
+  --daemon-address <rpc-address> \ 
+  --home <path>
+```
+
+Parameters:
+- `<fp-eots-pk-hex>`: The EOTS public key of the finality provider you want to 
+register (e.g., `cf0f03b9ee2d4a0f27240e2d8b8c8ef609e24358b2eb3cfd89ae4e4f472e1a41`)
+- `--daemon-address`: RPC address of the finality provider daemon
+  (default: `127.0.0.1:12581`)
+- `--home`: Path to your finality provider daemon home directory (e.g., `~/.fpdHome`)
+
+If successful, the command will return a transaction hash:
+
+``` shell
+{ "tx_hash":
+"C08377CF289DF0DC5FA462E6409ADCB65A3492C22A112C58EA449F4DC544A3B1" } 
+```
+
+You can verify the transaction was successful by looking up this transaction 
+hash on the Babylon chain.
```

Comment:
Should we provide a link to a third party explorer or CLI cmd to query the tx content?

Reply:
Don't think we should link to something here, not the purpose of this guide. This is just a small tip.

I think including the below TODO should be enough in terms of verification